symfony/plugins/orangehrmCorePlugin/Api/V2/Validator/Rules/EntityUniquePropertyOption.php

```diff
@@ -76,4 +78,42 @@ public function setTrimFunction(Closure $trimFunction): self
         $this->trimFunction = $trimFunction;
         return $this;
     }
+
+    /**
+     * @return bool
+     */
+    public function hasIgnoreValues(): bool
+    {
+        return isset($this->ignoreValues);
+    }
+
+    /**
+     * @param array $ignoreValues
+     * Getter method => value (if this value is set, entity is ignored)
+     * E.g: ['isDeleted' => true, 'getId' => 11]
+     */
+    public function setIgnoreValues(array $ignoreValues): void
+    {
+        $this->ignoreValues = $ignoreValues;
+    }
```

Comment:
```suggestion
    public function setIgnoreValues(array $ignoreValues): self
    {
        $this->ignoreValues = $ignoreValues;
        return $this;
    }
```
shall we go like other setter methods? it will be easy to call as chaining.

Reply:
setter was removed since the code was moved into `EntityUniqueProperty` class